Makefile

```diff
@@ -26,7 +26,7 @@ build: clean ## build files but do not run a server
 .PHONY: build
 
 check-html: build ## validate HTML
-	bundle exec htmlproofer --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/" --file-ignore "/.*\/files\/.*/" ./_site
+	bundle exec htmlproofer --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/","#" --file-ignore "/.*\/files\/.*/" ./_site
```

Comment:
theres an htmlproofer option for this too btw ` --allow-hash-href  If 'true', ignores the 'href' '#'`

Reply:
thanks! I did not check. Will change that